debug_toolbar/panels/sql/panel.py

```diff
@@ -149,7 +149,8 @@ def similar_key(query):
             return query['raw_sql']
 
         def duplicate_key(query):
-            return (query['raw_sql'], tuple(query['raw_params']))
+            raw_params = () if query['raw_params'] is None else tuple(query['raw_params'])
```

Comment:
Just ran into this as well using master, nice to get this fixed! 

I'd use though `tuple(query['raw_params'] or ())`

Reply:
@jieter: I prefer the more explicit `None` check, since `or` can hide unexpected type errors that should be raised rather than silenced.

Reply:
fine of course! currently working on this branch, works like charm!